nights/2024-05-29-README.md

```diff
@@ -1,16 +1,14 @@
 # 2024-05-29
 
-## Reviewed by:   your_name_here
+## Reviewed by:  Jacob Mailhot
```

Comment:
Even if there are no science images from the night we still have to delete the sections that are examples and fill out the rest of the form for consistency.